### PR TITLE
Document custom error page restrictions.

### PR DESCRIPTION
**Pull request by timoreimann**
@containous/traefik PTAL.

In particular, someone with sufficient expertise please acknowledge that custom error pages can really be used with the key/value stores we support.

Refs #2035.

### CODE REVIEW

File: docs/configuration/commons.md [hunk 1]
```diff
@@ -242,6 +242,8 @@ Instead, the query parameter can also be set to some generic error page like so:
 Now the `500s.html` error page is returned for the configured code range.
 The configured status code ranges are inclusive; that is, in the above example, the `500s.html` page will be returned for status codes `500` through, and including, `599`.
 
+Custom error pages are easiest to implement using the file provider or one of the supported K/V stores. For other dynamic providers, the corresponding template file needs to be customized accordingly and referenced in the Traefik configuration.
```
Comment (ldez) [inline docs/configuration/commons.md]: Could you add one sentence by line?

Comment (timoreimann) [reply docs/configuration/commons.md]: Done.

**Comment by Juliens** [comment]
In fact, all the k/v store use kv provider

```go
type Provider struct {
	kv.Provider `mapstructure:",squash"`
}
```
and the kv provider use template https://github.com/containous/traefik/blob/master/provider/kv/kv.go#L149

and  there is no status in the template

**Comment by timoreimann** [comment]
@Juliens thanks for digging into this. I updated the documentation.

Let me know what you think.

**Pull request merged**